packages/mobile/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@redwoodjs/mobile",
+  "version": "0.24.0",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "license": "MIT",
+  "dependencies": {
+    "@redwoodjs/internal": "^0.24.0",
+    "react-native": "^0.63.4"
```

Comment:
Wondering if there's any way that we can make this part of the "mobile" side instead of this package? Like we're doing for "react" and "reactDOM" in the web side.

Reply:
Not sure if I follow. What'd entail making it part of the mobile side?

Reply:
As a general rule, we minimize handling packages via the project template codebase -- the preference is to use Redwood packages to manage dependencies. BUT one example of an exception is here:
https://github.com/redwoodjs/redwood/blob/2f44c51217eca65f4e2ca8c2679a50918ff62deb/packages/create-redwood-app/template/web/package.json#L19-L21

So I believe Peter is referring to whether or not we should include in template codebase `./mobile/package.json`. 

Note: we refer to `./web/`, `./api/`, and `./mobile/` as Redwood Sides.

Reply:
Oh I see. I can follow your suggestion. However, it's worth mentioning that React Native is not great at following semantic versioning and it's common to see breaking changes being introduced. That could lead to developers updating those dependencies and breaking the contract that Redwood has with it. Thus my inclination towards pinning it from the package.